packages/ember-runtime/tests/system/core_object_test.js

```diff
@@ -93,6 +93,28 @@ moduleFor(
       getOwner(proxy);
     }
 
+    ['@test `Object.keys` works properly'](assert) {
```

Comment:
Can you tweak this test title? I'd like to see it reference the original issue, and specifically what it is intended to test.

Something like this seems reasonable:

```
@test observed properties are enumerable when set GH#14594
```

Also, can you move this to two tests one in `packages/ember-metal/tests/accessors/set_test.js` and another in `packages/ember-metal/tests/set_properties_test.js`?

Reply:
sure